docs/concepts/governance.md

```diff
@@ -244,6 +244,17 @@ There are certain parameters within Vega that influence the behaviour of the sys
 Network parameters can only be added and removed with Vega core software releases.
 
 :::note Go deeper
+* [Concepts: Network parameters](../concepts/vega-chain/network.md#parameters)
 * [Network parameters: See full list on the block explorer ↗](https://explorer.fairground.wtf/network-parameters)
 * [Tutorial: Propose a network parameter change](../tutorials/proposals/network-parameter-proposal.md)
 :::
+
+### Suggested ranges for parameters
+Some network parameters have minimum/maximum boundaries to ensure they are usefully functional. The table below contains those parameters, to be used as guidance when proposing changes to any of those parameters.
```

Comment:
I was assured the grammar is correct, but it reads odd. Also, I would see it more as non-nonsensical (such as requiring more than 100% votes), for some parameters we still allow pretty useless values (such as exactly 100% votes)